authorization-service/handlers/basicAuthorizer.js

```diff
@@ -0,0 +1,39 @@
+const generatePolicy = (principalId, effect, resource) => {
+  return {
+    principalId,
+    policyDocument: {
+      Version: "2012-10-17",
+      Statement: [
+        { Action: "execute-api:Invoke", Effect: effect, Resource: resource },
+      ],
+    },
+  };
+};
+
+exports.handler = async function (event, _context, callback) {

```

Comment:
There is 500 error with wrong auth token call 

<img width="899" alt="Screenshot 2025-03-26 at 10 17 20" src="https://github.com/user-attachments/assets/0146e6d3-1614-48fe-bb1e-6fa164e97c45" />